package.json

```diff
@@ -93,7 +93,7 @@
     "@angular/platform-browser-dynamic": "13.3.9",
     "@angular/router": "13.3.9",
     "@fontsource/roboto": "4.2.1",
-    "@ionic/angular": "^5.5.2",
+    "@ionic/angular": "6.1.11",
```

Comment:
Is there a reason for not having the same version here as in `libs/designsystem/package.json`? (`^6.1.7`)

Reply:
No, I forgot to update it there as well.